llvm/utils/lit/lit/TestRunner.py

```diff
@@ -356,7 +356,7 @@ def executeBuiltinPopd(cmd, shenv):
 def executeBuiltinExport(cmd, shenv):
     """executeBuiltinExport - Set an environment variable."""
     if len(cmd.args) != 2:
-        raise InternalShellError("'export' supports only one argument")
+        raise InternalShellError(cmd, "'export' supports only one argument")
```

Comment:
There should be an accompanying test.

Reply:
I created the test in my latest change. I only made a test to show the failing behavior that is used here, but I wasn't sure if I should make tests to show that the export command works. There are other lit tests that use the export command, which do show to some extent that the command works. Do you think that's enough, or should I add more tests?

Reply:
Your change literally only affects the error message, so I don't think that's required. As long as other tests are passing/failing the same as before your patch then I don't think you need to update the world. Now, it would be nice to improve the state of testing, but that shouldn't block you from making the error consistent.